upgrading/automated_upgrades.adoc

```diff
@@ -628,11 +628,11 @@ domain name.
 
 . Add any other `openshift_metrics_*` variables that you want to specify to
 override the defaults, as described in
-xref:../../install_config/cluster_metrics.adoc#metrics-ansible-variables[Specifying
+xref:../install_config/cluster_metrics.adoc#metrics-ansible-variables[Specifying
 Metrics Ansible Variables].
 
 . When you have finished updating your inventory file, follow the instructions in
-xref:../../install_config/cluster_metrics.html#deploying-the-metrics-components[Deploying the Metrics Deployment] to run the *_openshift_metrics.yml_* playbook and
+xref:../install_config/cluster_metrics.html#deploying-the-metrics-components[Deploying the Metrics Deployment] to run the *_openshift_metrics.yml_* playbook and
```

Comment:
Deploying the Metrics Deployment link is broken (s/.html/.adoc)

Reply:
Verifying the Upgrade is shared between Automated and Manual, but one has an extra step. Is that step missing from the other?

Reply:
Fixt